samples/client/petstore/java/resttemplate-withXml/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -597,8 +597,7 @@ public String generateQueryUri(MultiValueMap<String, String> queryParams, Map<St
                         queryBuilder.append(encodedName);
                         if (value != null) {
                             String templatizedKey = encodedName + valueItemCounter++;
-                            final String encodedValue = URLEncoder.encode(value.toString(), "UTF-8");
-                            uriParams.put(templatizedKey, encodedValue);
+                            uriParams.put(templatizedKey, value.toString());
```

Comment:
make sense to remove try-catch which was added specifically for `UrlEncoder.encode`

Reply:
There is use of `URLEncoder.encode` for the query parameter name (line 585)

Reply:
just noticed. thanks